deploy/README.md

```diff
@@ -88,8 +88,9 @@ The following matrix displays the tested package versions for our Helm chart.
 
 | Sumo Logic Helm Chart | kube-prometheus-stack/Prometheus Operator | FluentD | Fluent Bit                          | Falco  | Metrics Server | Telegraf Operator | Tailing Sidecar Operator |
 |:----------------------|:------------------------------------------|:--------|:------------------------------------|:-------|:---------------|:------------------|:-------------------------|
-| latest                | 12.10.0                                   | 1.12.2  | 0.14.1                              | 1.16.2 | 5.8.4          | 1.3.3             | 0.3.1                    |
-| 2.4.0 - 2.5.2         | 12.10.0                                   | 1.12.2  | 0.12.1                              | 1.16.2 | 5.8.4          | 1.3.3             | 0.3.1                    |
+| 2.6.0                 | 12.10.0                                   | 1.12.4  | 0.14.1                              | 1.16.2 | 5.8.4          | 1.3.3             | 0.3.2                    |
```

Comment:
This should be: `2.6.0 - latest` as we don't want to change this table if there are not dependency changes.